servers/zms/src/main/java/com/yahoo/athenz/zms/store/impl/jdbc/JDBCConnection.java

```diff
@@ -2603,6 +2618,32 @@ public boolean insertPolicy(String domainName, Policy policy) {
         return (affectedRows > 0);
     }
 
+    @Override
```

Comment:
Why do we have a separate API just for insertPolicyVersion? Isn't this the same as insertPolicy where policy.version is not null? So we can just update insertPolicy to make handle the version number accordingly. In the DB Service code we can make sure whenever we need to call just insertPolicy the version is set to null and we don't take the value from the user

Reply:
Fixed